bin/plugin/commands/changelog.js

```diff
@@ -501,8 +635,36 @@ async function getChangelog( settings ) {
 			continue;
 		}
 
+		const featureGroups = groupBy( groupPullRequests, getIssueFeature );
 		changelog += '### ' + group + '\n\n';
-		groupEntries.forEach( ( entry ) => ( changelog += entry + '\n' ) );
+
+		Object.keys( featureGroups )
+			.sort( ( a, b ) => {
+				// sort "Unknown" to always be at the end
+				if ( a === 'Unknown' ) {
+					return 1;
+				} else if ( b === 'Unknown' ) {
+					return -1;
+				}
+				return featureGroups[ b ].length - featureGroups[ a ].length;
+			} )
```

Comment:
I'd like to encapsulate this into a dedicated function. Requires too much reading of the code. Not self documenting.